### PR TITLE
Updated data scripts

### PR DESCRIPTION
**Pull request by samussiah**
@rbudrevich @ekgoodman @bzkrouse , all updated data files and scripts are in this repo, specifically in [safetyData](https://github.com/RhoInc/viz-library/tree/updated-data-scripts/examples/0000-sample-data/safetyData) and [safetyData-queries](https://github.com/RhoInc/viz-library/tree/updated-data-scripts/examples/0000-sample-data/safetyData-queries).

### CODE REVIEW

**Comment by jwildfire** [comment]
@samussiah Looks reasonable to me. The diffs are massive (and thus hard to follow) for some reason, but it seems like the main change is adding a seed, and changing up the way the random values are generated a bit. Are there any column names changing in the generated files?

**Comment by samussiah** [comment]
Nope, no column name changes.  A bunch of files were updated by `npm run build` before I even updated anything.
